agr_java_core/src/main/java/org/alliancegenome/neo4j/repository/VariantRepository.java

```diff
@@ -33,13 +33,13 @@ public List<Variant> getVariantsOfAllele(String id) {
 		query += " WITH a, variant, p1, synonyms, notes, pubs, crossRefs, collect(consequence) AS consequences";
 		query += " OPTIONAL MATCH transcripts=(:GenomicLocation)--(variant:Variant)-[:ASSOCIATION]-(t:Transcript)<-[:TRANSCRIPT_TYPE]-(:SOTerm)";
 		query += " WITH a, variant, p1, synonyms, notes, pubs, crossRefs, consequences, collect(transcripts) AS transcripts";
-		query += " OPTIONAL MATCH transcriptConsequence=(variant:Variant)--(tlc:TranscriptLevelConsequence)<-[:ASSOCIATION]-(t:Transcript)--(variant:Variant)";
-		query += " WITH a, variant, p1, synonyms, notes, pubs, crossRefs, consequences, transcripts, collect(transcriptConsequence) AS transcriptConsequences";
+		query += " OPTIONAL MATCH transcriptConsequences=(variant:Variant)--(tlc:TranscriptLevelConsequence)<-[:ASSOCIATION]-(t:Transcript)--(variant:Variant)";
+		query += " WITH a, variant, p1, synonyms, notes, pubs, crossRefs, consequences, transcripts, collect(transcriptConsequences) AS transcriptConsequences";
 		query += " OPTIONAL MATCH loc=(variant:Variant)-[:ASSOCIATION]->(:GenomicLocation)";
 		query += " WITH a, variant, p1, synonyms, notes, pubs, crossRefs, consequences, transcripts, transcriptConsequences, collect(loc) AS locs";
 		query += " OPTIONAL MATCH p2=(variant:Variant)<-[:COMPUTED_GENE]-(:Gene)-[:ASSOCIATION]->(:GenomicLocation)";
 		query += " WITH p1, collect(p2) AS p2, locs, consequences, synonyms, transcripts, transcriptConsequences, notes, crossRefs, pubs";
-		query += " RETURN p1, p2, locs, consequences, synonyms, transcripts, transcriptConsequence, notes, crossRefs, pubs ";
+		query += " RETURN p1, p2, locs, consequences, synonyms, transcripts, transcriptConsequences, notes, crossRefs, pubs ";
```

Comment:
It should be just the `s` added to the return line... no need to change the prior entries